reana_workflow_controller/rest/utils.py

```diff
@@ -125,6 +127,18 @@ def get_workflow_name(workflow):
     return workflow.name + "." + str(workflow.run_number)
 
 
+def get_workflow_status_name(workflow):
```

Comment:
Have you tried to move such logic into a new property in REANA-DB? (asking just in case you already found a blocker 😄 ). We could have something like `Workflow.status_ui` (this name is bad, this is just a placeholder) which could be a [hybrid property](https://docs.sqlalchemy.org/en/14/orm/extensions/hybrid.html). This way we could perhaps overwrite the `.in_` method for that property (useful for [filtering](https://github.com/reanahub/reana-workflow-controller/blob/24c3a6222c09bb282ac0153e930077be292eee0f/reana_workflow_controller/rest/workflows.py#L228)), and also we wouldn't break the UI nor the client ([this](https://github.com/reanahub/reana-workflow-controller/blob/24c3a6222c09bb282ac0153e930077be292eee0f/reana_workflow_controller/rest/workflows.py#L237) would access hypothetical `Workflow.status_ui`).

On the one hand, if we do such a split I feel it should be close to the models, on the other hand, I see it might be confusing to decide which one to use. Let me know what you think.

Reply:
Created issue to follow up on this [here](https://github.com/reanahub/reana-workflow-controller/issues/375)